lib/omnibus/project.rb

```diff
@@ -913,6 +947,11 @@ def build_me
       # Cache the build order so we don't re-compute
       softwares = library.build_order
 
+      # Install any package this project extends
+      extended_packages.each do |package, enablerepo|
+        packager.install(package, enablerepo)
```

Comment:
here, calling `install()` will run `apt-get update` multiple times which is probably going to end up being very slow.
Also you could pass to `install()` a list of packages so that they can be installed in one-shot.